.github/workflows/update-otel.yaml

```diff
@@ -28,6 +28,6 @@ jobs:
           make genotelcontribcol
           make update-otel
           git push --set-upstream origin $branch  
-          gh pr create --base main --title "[chore] Prepare release ${CANDIDATE_BETA}" --body "This PR updates the opentelemetry-collector dependency to the latest release"
+          gh pr create --base main --title "[chore] Update core dependencies ${CANDIDATE_BETA}" --body "This PR updates the opentelemetry-collector dependency to the latest release"
```

Comment:
lol it's my bad copy/paste. Thanks for updating. `${CANDIDATE_BETA}` isn't needed either https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37401